src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMA = ",";
+    private static final String NOTWORD = "\\W+";
+    private static int supply;
+    private static int buy;
+    private static int result;
+    private StringBuilder builderWordsAll = new StringBuilder();
```

Comment:
It's better to create new StringBuilder() in method, not one instance for whole class

Reply:
How do it correct? I use value StringBuilder from readFromFile for createReport. And I need have value in class

Reply:
You need to return the string from the readFromFile() method and add it as a parameter to the createReport(String data) method, and pass it there